db/migrate/201509151000_create_qualifying_comments_table.rb

```diff
@@ -0,0 +1,12 @@
+class CreateQualifyingCommentsTable < ActiveRecord::Migration
+  def change
+  	create_table :qualifyings do |t|
+      t.integer :user_id, null: false
+      t.integer :num_qualifying, null: false
```

Comment:
If you're in a table named "qualifyings" then you don't need the "qualifying" suffix.

If the whole table is about counts, then let's call this `total`.